warzone/Strategies.cpp

```diff
@@ -1,4 +1,4 @@
-#include "Strategy.h";
+#include "Strategies.h";
```

Comment:
Crap, apparently it's supposed to be `PlayerStrategies.h` instead >.<

Reply:
Fixed in 452b73d